ssh/main.go

```diff
@@ -22,6 +23,13 @@ const (
 )
 
 func main() {
+	if value, ok := os.LookupEnv("SHELLHUB_ENV"); ok && value == "development" {
+		log.SetLevel(log.TraceLevel)
```

Comment:
One aspect to consider is if we should allow setting the log level directly. Even in production, putting it to trace may help when debugging issues.